### ISSUE TITLE
"ERROR ~ No such variable: colors" when running the demo pipeline

### ISSUE THREAD

**Issue by vmkalbskopf** [epi2me-labs/wf-human-variation#208]
### Operating System

Ubuntu 22.04

### Workflow Version

v2.3.1-g6a6daa6

### Workflow Execution

Command line (Local)

### CLI command run

nextflow run epi2me-labs/wf-human-variation \
	--bam 'wf-human-variation-demo/demo.bam' \
	--ref 'wf-human-variation-demo/demo.fasta' \
	--bed 'wf-human-variation-demo/demo.bed' \
	--sample_name 'DEMO' \
	--snp \
	--sv \
	--mod \
	--phased \
	-profile standard

### Workflow Execution - CLI Execution Profile

standard (default)

### What happened?

I am running the current version of Nextflow: 24.04.4

This is the first time I am running this pipeline, and I have not installed the desktop EPI2ME app on this computer.

When I run the command, it failes with the error `"ERROR ~ No such variable: colors"`

### Relevant log output

```shell
Aug-09 11:28:41.304 [main] DEBUG nextflow.cli.Launcher - $> nextflow run epi2me-labs/wf-human-variation --bam wf-human-variation-demo/demo.bam --ref wf-human-variation-demo/demo.fasta --bed wf-human-variation-demo/demo.bed --sample_name DEMO --snp -
-sv --mod --phased
Aug-09 11:28:41.386 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 24.04.4
Aug-09 11:28:41.405 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/home/bioinformatics/.nextflow/plugins; core-plugins: nf-amazon@2.5.3,nf-azure@1.6.1,nf-cloudcache@0.4.1,nf-codecommi
t@0.2.1,nf-console@1.1.3,nf-ga4gh@1.3.0,nf-google@1.13.2-patch1,nf-tower@1.9.1,nf-wave@1.4.2-patch1
Aug-09 11:28:41.416 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Aug-09 11:28:41.417 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Aug-09 11:28:41.419 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.12.0 in 'deployment' mode
Aug-09 11:28:41.431 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Aug-09 11:28:41.444 [main] DEBUG nextflow.scm.ProviderConfig - Using SCM config path: /home/bioinformatics/.nextflow/scm
Aug-09 11:28:42.129 [main] DEBUG nextflow.scm.AssetManager - Git config: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-human-variation.git
Aug-09 11:28:42.157 [main] DEBUG nextflow.scm.RepositoryFactory - Found Git repository result: [RepositoryFactory]
Aug-09 11:28:42.167 [main] DEBUG nextflow.scm.AssetManager - Git config: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-human-variation.git
Aug-09 11:28:42.934 [main] DEBUG nextflow.config.ConfigBuilder - Found config base: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.config
Aug-09 11:28:42.947 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.config
Aug-09 11:28:42.969 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /home/bioinformatics/.nextflow/secrets/store.json
Aug-09 11:28:42.975 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@58ae402b] - activable => nextflow.secret.LocalSecretsProvider@58ae402b
Aug-09 11:28:42.998 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `standard`
Aug-09 11:28:41.304 [main] DEBUG nextflow.cli.Launcher - $> nextflow run epi2me-labs/wf-human-variation --bam wf-human-variation-demo/demo.bam --ref wf-human-variation-demo/demo.fasta --bed wf-human-variation-demo/demo.bed --sample_name DEMO --snp -
-sv --mod --phased
Aug-09 11:28:41.386 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 24.04.4
Aug-09 11:28:41.405 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/home/bioinformatics/.nextflow/plugins; core-plugins: nf-amazon@2.5.3,nf-azure@1.6.1,nf-cloudcache@0.4.1,nf-codecommi
t@0.2.1,nf-console@1.1.3,nf-ga4gh@1.3.0,nf-google@1.13.2-patch1,nf-tower@1.9.1,nf-wave@1.4.2-patch1
Aug-09 11:28:41.416 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Aug-09 11:28:41.417 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Aug-09 11:28:41.419 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.12.0 in 'deployment' mode
Aug-09 11:28:41.304 [main] DEBUG nextflow.cli.Launcher - $> nextflow run epi2me-labs/wf-human-variation --bam wf-human-variation-demo/demo.bam --ref wf-human-variation-demo/demo.fasta --bed wf-human-variation-demo/demo.bed --sample_name DEMO --snp -
-sv --mod --phased
Aug-09 11:28:41.386 [main] DEBUG nextflow.cli.CmdRun - N E X T F L O W  ~  version 24.04.4
Aug-09 11:28:41.405 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; embedded=false; plugins-dir=/home/bioinformatics/.nextflow/plugins; core-plugins: nf-amazon@2.5.3,nf-azure@1.6.1,nf-cloudcache@0.4.1,nf-codecommi
t@0.2.1,nf-console@1.1.3,nf-ga4gh@1.3.0,nf-google@1.13.2-patch1,nf-tower@1.9.1,nf-wave@1.4.2-patch1
Aug-09 11:28:41.416 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Aug-09 11:28:41.417 [main] INFO  o.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Aug-09 11:28:41.419 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.12.0 in 'deployment' mode
Aug-09 11:28:41.431 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Aug-09 11:28:41.444 [main] DEBUG nextflow.scm.ProviderConfig - Using SCM config path: /home/bioinformatics/.nextflow/scm
Aug-09 11:28:42.129 [main] DEBUG nextflow.scm.AssetManager - Git config: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-human-variation.git
Aug-09 11:28:42.157 [main] DEBUG nextflow.scm.RepositoryFactory - Found Git repository result: [RepositoryFactory]
Aug-09 11:28:42.167 [main] DEBUG nextflow.scm.AssetManager - Git config: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/.git/config; branch: master; remote: origin; url: https://github.com/epi2me-labs/wf-human-variation.git
Aug-09 11:28:42.934 [main] DEBUG nextflow.config.ConfigBuilder - Found config base: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.config
Aug-09 11:28:42.947 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.config
Aug-09 11:28:42.969 [main] DEBUG n.secret.LocalSecretsProvider - Secrets store: /home/bioinformatics/.nextflow/secrets/store.json
Aug-09 11:28:42.975 [main] DEBUG nextflow.secret.SecretsLoader - Discovered secrets providers: [nextflow.secret.LocalSecretsProvider@58ae402b] - activable => nextflow.secret.LocalSecretsProvider@58ae402b
Aug-09 11:28:42.998 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `standard`
Aug-09 11:28:43.411 [main] DEBUG nextflow.cli.CmdRun - Applied DSL=2 from script declaration
Aug-09 11:28:43.412 [main] DEBUG nextflow.cli.CmdRun - Launching `https://github.com/epi2me-labs/wf-human-variation` [ecstatic_elion] DSL2 - revision: 6a6daa65aa [master]
Aug-09 11:28:43.413 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
Aug-09 11:28:43.413 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins resolved requirement=[]
Aug-09 11:28:43.513 [main] DEBUG nextflow.Session - Session UUID: d426c8f1-fdbe-4d5c-99b1-547df59f9286
Aug-09 11:28:43.514 [main] DEBUG nextflow.Session - Run name: ecstatic_elion
Aug-09 11:28:43.515 [main] DEBUG nextflow.Session - Executor pool size: 48
Aug-09 11:28:43.529 [main] DEBUG nextflow.file.FilePorter - File porter settings maxRetries=3; maxTransfers=50; pollTimeout=null
Aug-09 11:28:43.538 [main] DEBUG nextflow.util.ThreadPoolBuilder - Creating thread pool 'FileTransfer' minSize=10; maxSize=144; workQueue=LinkedBlockingQueue[10000]; allowCoreThreadTimeout=false
Aug-09 11:28:43.564 [main] DEBUG nextflow.cli.CmdRun -
  Version: 24.04.4 build 5917
  Created: 01-08-2024 07:05 UTC (09:05 CEST)
  System: Linux 6.5.0-45-generic
  Runtime: Groovy 4.0.21 on OpenJDK 64-Bit Server VM 11.0.24+8-post-Ubuntu-1ubuntu322.04
  Encoding: UTF-8 (UTF-8)
  Process: 1030515@nextron-X11DPi-N-T [127.0.1.1]
  CPUs: 48 - Mem: 251.4 GB (1 GB) - Swap: 2 GB (2 GB)
Aug-09 11:28:43.589 [main] DEBUG nextflow.Session - Work-dir: /home/bioinformatics/epi2me_testing/work [ext2/ext3]
Aug-09 11:28:43.629 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
Aug-09 11:28:43.640 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Aug-09 11:28:43.708 [main] DEBUG nextflow.cache.CacheFactory - Using Nextflow cache factory: nextflow.cache.DefaultCacheFactory
Aug-09 11:28:43.718 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 49; maxThreads: 1000
Aug-09 11:28:43.772 [main] DEBUG nextflow.Session - Session start
Aug-09 11:28:43.775 [main] DEBUG nextflow.trace.TraceFileObserver - Workflow started -- trace file: /home/bioinformatics/epi2me_testing/[:]/execution/trace.txt
Aug-09 11:28:43.781 [main] DEBUG nextflow.Session - Using default localLib path: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/lib
Aug-09 11:28:43.784 [main] DEBUG nextflow.Session - Adding to the classpath library: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/lib
Aug-09 11:28:43.785 [main] DEBUG nextflow.Session - Adding to the classpath library: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/lib/nfcore_external_java_deps.jar
Aug-09 11:28:45.766 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Aug-09 11:28:52.389 [main] INFO  nextflow.Nextflow -
ESC[0;92m||||||||||   ESC[0mESC[2m_____ ____ ___ ____  __  __ _____      _       _
ESC[0;92m||||||||||  ESC[0mESC[2m| ____|  _ \_ _|___ \|  \/  | ____|    | | __ _| |__  ___
ESC[0;33m|||||       ESC[0mESC[2m|  _| | |_) | |  __) | |\/| |  _| _____| |/ _` | '_ \/ __|
ESC[0;33m|||||       ESC[0mESC[2m| |___|  __/| | / __/| |  | | |__|_____| | (_| | |_) \__ \
ESC[0;94m||||||||||  ESC[0mESC[2m|_____|_|  |___|_____|_|  |_|_____|    |_|\__,_|_.__/|___/
ESC[0;94m||||||||||  ESC[0mESC[1mwf-human-variation v2.3.1-g6a6daa6ESC[0m
ESC[2m--------------------------------------------------------------------------------ESC[0m
ESC[1mCore Nextflow optionsESC[0m
  ESC[0;34mrevision       : ESC[0;32mmasterESC[0m
  ESC[0;34mrunName        : ESC[0;32mecstatic_elionESC[0m
  ESC[0;34mcontainerEngine: ESC[0;32mdockerESC[0m
  ESC[0;34mcontainer      : ESC[0;32m[withLabel:wf_human_snp:ontresearch/wf-human-variation-snp:sha17e686336bf6305f9c90b36bc52ff9dd1fa73ee9, withLabel:wf_human_sv:ontresearch/wf-human-variation-sv:shac591518dd32ecc3936666c95ff08f6d7474e9728, withLab
el:wf_human_mod:ontresearch/modkit:shae137452eb41f5f12b790774cafe15bc97f48d4d0, withLabel:wf_cnv:ontresearch/wf-cnv:sha428cb19e51370020ccf29ec2af4eead44c6a17c2, withLabel:wf_human_str:ontresearch/wf-human-variation-str:shadd2f2963fe39351d4e0d6fa3ca5
4e1064c6ec057, withLabel:snpeff_annotation:ontresearch/snpeff:shadcc812849019640d4d2939703fbb8777256e41ad, withLabel:wf_common:ontresearch/wf-common:shad399cf22079b5b153920ac39ee40095a677933f1, withLabel:spectre:ontresearch/spectre:sha49a9fe474da986
0f84f08f17f137b47a010b1834, default:ontresearch/wf-human-variation:sha2b856c1f358ddf1576217a336bc0e9864b6dc0ed]ESC[0m
  ESC[0;34mlaunchDir      : ESC[0;32m/home/bioinformatics/epi2me_testingESC[0m
  ESC[0;34mworkDir        : ESC[0;32m/home/bioinformatics/epi2me_testing/workESC[0m
  ESC[0;34mprojectDir     : ESC[0;32m/home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variationESC[0m
  ESC[0;34muserName       : ESC[0;32mbioinformaticsESC[0m
  ESC[0;34mprofile        : ESC[0;32mstandardESC[0m
  ESC[0;34mconfigFiles    : ESC[0;32m/home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/nextflow.configESC[0m

ESC[1mWorkflow OptionsESC[0m
  ESC[0;34msv             : ESC[0;32mtrueESC[0m
  ESC[0;34msnp            : ESC[0;32mtrueESC[0m
  ESC[0;34mmod            : ESC[0;32mtrueESC[0m

ESC[1mMain optionsESC[0m
  ESC[0;34msample_name    : ESC[0;32mDEMOESC[0m
  ESC[0;34mbam            : ESC[0;32mwf-human-variation-demo/demo.bamESC[0m
  ESC[0;34mref            : ESC[0;32mwf-human-variation-demo/demo.fastaESC[0m
  ESC[0;34mbed            : ESC[0;32mwf-human-variation-demo/demo.bedESC[0m
  ESC[0;34mphased         : ESC[0;32mtrueESC[0m

!! Only displaying parameters that differ from the pipeline defaults !!
ESC[2m--------------------------------------------------------------------------------ESC[0m
If you use epi2me-labs/wf-human-variation for your analysis please cite:

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x


ESC[2m--------------------------------------------------------------------------------ESC[0m
This is epi2me-labs/wf-human-variation v2.3.1-g6a6daa6.
ESC[2m--------------------------------------------------------------------------------ESC[0m
Aug-09 11:28:52.420 [main] DEBUG nextflow.script.ScriptRunner - Parsed script files:
  Script_d79719a7db092e3d: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/wf-human-cnv-qdnaseq.nf
  Script_08fc0a10300c9cc7: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/wf-human-cnv.nf
  Script_ea72d6b94b65cd57: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/wf-human-cnv-qdnaseq.nf
  Script_a65eda53e4c60c74: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/main.nf
  Script_4e4ab5d902a6aef7: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/wf-human-str.nf
  Script_c3da32edbd8e31d3: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/wf-human-sv.nf
  Script_55c8fdc6f8caabcf: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./modules/local/wf-human-sv.nf
  Script_0fc1aa781c46dbd1: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/wf-human-snp.nf
  Script_109684392c51576a: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/common.nf
  Script_3297b401abf5ff91: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/wf-human-str.nf
  Script_82427989a69e26b0: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./lib/_ingress.nf
  Script_f97f3346df503fce: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/wf-human-snp.nf
  Script_b6fd3c4ddd1b8546: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./lib/./ingress.nf
  Script_dbad131e46599a28: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/wf-human-sv-eval.nf
  Script_9742aaac25bcf521: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./lib/reference.nf
  Script_02975dbf6c362ffe: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/../modules/local/wf-human-cnv.nf
  Script_8eb7f6cadf8cafa0: /home/bioinformatics/.nextflow/assets/epi2me-labs/wf-human-variation/./workflows/methyl.nf
Aug-09 11:28:52.421 [main] DEBUG nextflow.Session - Session aborted -- Cause: No such property: colors for class: nextflow.script.WorkflowBinding
Possible solutions: class
Aug-09 11:28:52.724 [main] ERROR nextflow.cli.Launcher - @unknown
groovy.lang.MissingPropertyException: No such property: colors for class: nextflow.script.WorkflowBinding
Possible solutions: class
        at groovy.lang.Binding.getVariable(Binding.java:61)
        at nextflow.script.WorkflowBinding.getVariable(WorkflowBinding.groovy:141)
        at groovy.lang.Binding.getProperty(Binding.java:117)
        at nextflow.script.WorkflowBinding.getProperty(WorkflowBinding.groovy:130)
        at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:167)
        at groovy.lang.Closure.getPropertyTryThese(Closure.java:325)
        at groovy.lang.Closure.getPropertyDelegateFirst(Closure.java:315)
        at groovy.lang.Closure.getProperty(Closure.java:301)
        at org.codehaus.groovy.vmplugin.v8.IndyInterface.fromCache(IndyInterface.java:321)
        at Script_9742aaac25bcf521$_runScript_closure5$_closure11.doCall(Script_9742aaac25bcf521:100)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:343)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:328)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:279)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1007)
        at groovy.lang.Closure.call(Closure.java:433)
        at groovy.lang.Closure.call(Closure.java:412)
        at nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:204)
        at nextflow.script.WorkflowDef.run(WorkflowDef.groovy:188)
        at nextflow.script.BindableDef.invoke_a(BindableDef.groovy:51)
        at nextflow.script.ComponentDef.invoke_o(ComponentDef.groovy:40)
        at nextflow.script.WorkflowBinding.invokeMethod(WorkflowBinding.groovy:103)
        at org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:651)
        at org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:628)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeOnDelegationObjects(ClosureMetaClass.java:391)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:330)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1007)
        at org.codehaus.groovy.vmplugin.v8.IndyInterface.fromCache(IndyInterface.java:321)
        at Script_a65eda53e4c60c74$_runScript_closure1$_closure4.doCall(Script_a65eda53e4c60c74:198)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:343)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:328)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:279)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1007)
        at groovy.lang.Closure.call(Closure.java:433)
        at groovy.lang.Closure.call(Closure.java:412)
        at nextflow.script.WorkflowDef.run0(WorkflowDef.groovy:204)
        at nextflow.script.WorkflowDef.run(WorkflowDef.groovy:188)
        at nextflow.script.BindableDef.invoke_a(BindableDef.groovy:51)
        at org.codehaus.groovy.vmplugin.v8.IndyInterface.fromCache(IndyInterface.java:321)
        at nextflow.script.BaseScript.run0(BaseScript.groovy:198)
        at org.codehaus.groovy.vmplugin.v8.IndyInterface.fromCache(IndyInterface.java:321)
        at nextflow.script.BaseScript.run(BaseScript.groovy:209)
        at nextflow.script.ScriptParser.runScript(ScriptParser.groovy:236)
        at nextflow.script.ScriptRunner.run(ScriptRunner.groovy:243)
        at nextflow.script.ScriptRunner.execute(ScriptRunner.groovy:138)
        at nextflow.cli.CmdRun.run(CmdRun.groovy:372)
        at nextflow.cli.Launcher.run(Launcher.groovy:503)
        at nextflow.cli.Launcher.main(Launcher.groovy:657)
```

### Were you able to successfully run the latest version of the workflow with the demo data?

no

**Comment by vmkalbskopf** [comment]
The error is occuring when the lib/reference.nf script runs to prep the reference file. The fact that the function to check if if the ref exists fails means that it can't find the ref. Did I need to download the demo reference separately?

````
   if (file(margs.input_ref).exists()){
            ref = Channel.fromPath(margs.input_ref)
        } else {
            throw new Exception(colors.red + "File ${margs.input_ref} not found." + colors.reset)
        }
````

**Comment by vmkalbskopf** [comment]
Manually downloading the demo files from AWS and then specifying the files in the command seems to have fixed the problem. But I presume this is not the expected behaviour.

**Comment by RenzoTale88** [comment]
@vmkalbskopf thanks for notifying this, I'll open a ticket for it.

**Comment by RenzoTale88** [comment]
@vmkalbskopf this should be fixed as of v2.4.0. If you come across further issues with the workflow please do open a new github issue.

Andrea
